packages/core/src/common/performance/stopwatch.ts

```diff
@@ -154,4 +181,8 @@ export abstract class Stopwatch {
         this.logger.log(level, `${whatWasMeasured}: ${elapsed.toFixed(1)} ms [${timeFromStart}]`, ...(options.arguments ?? []));
     }
 
+    getCachedResults(): MeasurementResult[] {
```

Comment:
Why not use a property accessor returning a readonly array? It prevent unnecessary copying.